Author: Sven Joachim <svenjoac@gmx.de>
Description: Debian changes to xterm* terminfo entries
 The "rep" feature has been removed from xterm-new and derived terminfo
 descriptions, as it causes compatibility problems with older terminal
 emulators.
 .
 The xterm-debian terminfo entry is provided as an alias for xterm, as
 some ancient Debian systems used xterm-debian as $TERM.
Bug-Debian: https://bugs.debian.org/933053
Forwarded: not-needed
Last-Update: 2023-06-05

---
 misc/terminfo.src |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -5005,7 +5005,7 @@ xterm-p370|xterm patch #370,
 	kcbt=\E[Z, kent=\EOM, nel=\EE,
 	rv=\E\\[41;[1-6][0-9][0-9];0c,
 	xr=\EP>\\|XTerm\\([1-9][0-9]+\\)\E\\\\,
-	use=ecma+index, use=ansi+rep, use=ecma+strikeout,
+	use=ecma+index, use=ecma+strikeout,
 	use=xterm+pcfkeys, use=xterm+nofkeys,
 	use=bracketed+paste, use=report+version,
 	use=xterm+focus,
@@ -5681,7 +5681,7 @@ xterm-24|vs100|xterms|xterm terminal emu
 	lines#24, use=xterm-old,
 
 # This is xterm for ncurses.
-xterm|xterm terminal emulator (X Window System),
+xterm|xterm-debian|xterm terminal emulator (X Window System),
 	use=xterm-new,
 
 # This entry assumes that xterm's handling of VT100 SI/SO is disabled by
